### PR TITLE
git-clone: add clone single branch example

### PR DESCRIPTION
**Pull request by iNemesis**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/git-clone.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Clone an existing repository only fetching the 10 most recent commits on the default branch (useful to save time):
 
 `git clone --depth {{10}} {{remote_repository_location}}`
+
+- Clone a single branch:
+
+`git clone -b {{branch_to_clone}} --single-branch {{remote_repository_location}}`
```
Comment (navarroaxel) [inline pages/common/git-clone.md:28]: Please use the long argument format:
```suggestion
`git clone --branch {{name}} --single-branch {{remote_repository_location}}`
```

File: pages/common/git-clone.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - Clone an existing repository only fetching the 10 most recent commits on the default branch (useful to save time):
 
 `git clone --depth {{10}} {{remote_repository_location}}`
+
+- Clone a single branch:
```
Comment (navarroaxel) [inline pages/common/git-clone.md:26]: Based on the previous example description:
```suggestion
- Clone an existing repository only fetching a specific branch:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @iNemesis, welcome and thanks for your submission. I left 2 comments.

I think this page needs the basic example "clone into a specific directory":
```
git clone {{remote_repository_location}} {{path/to/directory}}
```
Could you add it?

File: pages/common/git-clone.md [hunk 3]
```diff
@@ -22,3 +22,11 @@
 - Clone an existing repository only fetching the 10 most recent commits on the default branch (useful to save time):
 
 `git clone --depth {{10}} {{remote_repository_location}}`
+
+- Clone an existing repository only fetching a specific branch:
+
+`git clone --branch {{name}} --single-branch {{remote_repository_location}}`
+
+- Clone into a specific repository:
```
Comment (navarroaxel) [inline pages/common/git-clone.md:30]: Cloud you move this example as example number 2?
 ```suggestion
- Clone an existing repository into a specific directory:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by iNemesis** [comment]
Hi @navarroaxel, thanks for your suggestions !

I have added the "clone into a repository" command.

Have a nice day ;)

**Review by navarroaxel — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good.

**Pull request merged**